develop/reference/articles/01-intro/00-development-reference-intro.markdown

```diff
@@ -116,16 +64,31 @@ models and create powerful UI components in your presentation layer.
 
 <p style="margin-left: 40px;">
 <span style="font-size:18px;">
-<a href="http://docs.liferay.com/portal/7.0/taglibs/" target="_blank">
-Portal Taglibs <span class="opens-new-window-accessible">(Opens New Window)</span>
+<a href="https://docs.liferay.com/portal/7.0/taglibs/" target="_blank">
+Portal Taglibs&nbsp;<span class="opens-new-window-accessible">(Opens New Window)</span>
+</a>
+</span>
+</p>
+
+<p style="margin-left: 40px;">
+Describes Liferay Portal 7.0 CE tag libraries at the time of the latest GA release. You can
+also download them from the 
+<a href="https://www.liferay.com/downloads" target="_blank">Downloads page<span
+class="opens-new-window-accessible"> (Opens New Window)</span></a>.
+</p>
+
+<p style="margin-left: 40px;">&nbsp;</p>
+
+<p style="margin-left: 40px;">
+<span style="font-size:18px;">
+<a href="https://docs.liferay.com/portal/7.0/javadocs/" target="_blank">
+Module Taglibs for All Versions&nbsp;<span class="opens-new-window-accessible">(Opens New Window)</span>
```

Comment:
Same as for Javadoc, although in this case it's easier since there are fewer: Can we list the modularized javadocs here?